components/reddit/sources/new-comments-by-user/new-comments-by-user.js

```diff
@@ -0,0 +1,95 @@
+const common = require("../../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-by-user",
+  name: "New comments by user",
+  description: "Emits an event each time a user posts a new comment.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    reddit,
```

Comment:
What's the need for this? `...common.props` should work fine:

```js
const common = require("../common");

module.exports = {
  ...common,
  props: {
    ...common.props,
    username: {
      propDefinition: [
        common.props.reddit,
        "username",
      ],
    },
  },
};
```

Reply:
I still needed the `const { reddit } = common.props;` but `reddit` as a prop was removed.